File: solutions/uri/1006/1006.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`);

    val a: String = sc.next();
    val b: String = sc.next();
    val c: String = sc.next();

    println(
        "MEDIA = %.1f".format(
            (a.toDouble() * 2.0 + b.toDouble() * 3.0 + c.toDouble() * 5.0) /
            10.0
        )
    );
}

File: solutions/uri/1006/1006.scala
object Main {
    def main(args: Array[String]) {
        val a = io.StdIn.readLine().toDouble
        val b = io.StdIn.readLine().toDouble
        val c = io.StdIn.readLine().toDouble

        println("MEDIA = %.1f".format((a * 2.0 + b * 3.0 + c * 5.0) / 10.0))
    }
}
